library/swift/src/StreamClientBuilder.swift

```diff
@@ -114,8 +114,9 @@ public final class StreamClientBuilder: NSObject {
   ///
```

Comment:
Can we document `filterName` with what we expect the user to specify (i.e., uniqueness)?

Reply:
Sure thing.